indico/modules/events/persons/templates/email_dialog.html

```diff
@@ -0,0 +1,17 @@
+{% from 'forms/_form.html' import simple_form %}
+{{ simple_form(form, submit=_('Send')) }}
+<script>
+    var emailCountLimit = 10;
+    var $recipients = $('#form-group-recipients').find('.form-field');
+    var count = ($recipients.html().match(/@/g) || []).length;
+    if (count > emailCountLimit) {
+        $recipients.addClass('recipients-truncate');
+        var $more = $("<a>", {class: "show-more"});
+        $more.html('Show more');
+        $more.on('click', function() {
+            $recipients.toggleClass('recipients-expand');
+            $more.html($more.text() == 'Show more' ? 'Show less' : 'Show more');
```

Comment:
checking the current text is quite ugly. why not check `hasClass('recipients-expand')` to know the state and thus which text to use?